src/sentry/middleware/proxy.py

```diff
@@ -153,6 +155,22 @@ def process_request(self, request):
             request._stream = io.BufferedReader(UWsgiChunkedInput())
             request.META["CONTENT_LENGTH"] = "4294967295"  # 0xffffffff
 
+    def process_response(self, request, response):
+        self._process_response_impl(request, response)
+        return response
+
+    def _process_response_impl(self, request, response):
```

Comment:
I would name this something like _record_metrics` but 🤷‍♂️

Reply:
you're
 right. i hope the code is gone soon anyway